app.models.js

```diff
@@ -8,6 +10,37 @@ const fetchTopics = () => {
   });
 };
 
+const fetchArticles = (queries) => {
+  const { sort_by, order } = queries;
+
+  const sortByGreenList = ["created_at"];
+  const orderbyGreenList = ["desc", "asc"];
+
+  const defaultSortBy = "created_at";
+  const defaultOrder = "desc";
+
+  let SQLString = `SELECT articles.*, COUNT(comments.article_id) AS comment_count 
```

Comment:
Well done! It's a beefy query, so lovely to see you do this. And very well done to include valid sort/order by